example-library/src/main/java/uk/gov/gchq/palisade/example/library/rule/EmployeePassThroughRule.java

```diff
@@ -21,9 +21,20 @@
 import uk.gov.gchq.palisade.user.User;
 import uk.gov.gchq.syntheticdatagenerator.types.Employee;
 
+/**
+ * A simple rule implementation that does not redact any record fields
+ */
 public class EmployeePassThroughRule implements Rule<Employee> {
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Applies the rule to a record
+     *
+     * @param record the record being processed
+     * @param user the user making the request
```

Comment:
Updated

Reply:
theres not much point adding a {@code User} tag unless the link requires some formatting that you dont want changed
either add it as a {@link User} if it can resolve the link. or leave it as:
`@param user the user making the request`

Reply:
These have been changed to `@link`s now